_posts/2021-03-04-geoserver-2-19-rc-released.md

```diff
@@ -0,0 +1,195 @@
+  * The rest of this blog post highlights new features for GeoServer 2.19, please try out these features, read the documentation links, and ask questions.
+
+Known Issues:
+
+  * Layer configured with missing style throws NPE
+
+## MapML extension
+
+The MapML plugin has been under developer for some time as a community module and is now ready for a wider audience.
+
+(example)
+
+(screen snap)
+
+(documentation link)
+
+(thanks to contributor and customer for this development)
+
+## WPS JDBC extension
+
+The WPS JDBC extension allows to share the status of asynchronous WPS requests across a GeoServer cluster. The status of 
+all requests, past and ongoing, can be stored in a database, for later reference.
+
+The module uses GeoTools JDBC stores to access databases, create the necessary tables, and track status. 
+Connection parameters are provided as property files, e.g.:
+
+
+```
+user=postgres
+port=5432
+password=******
+passwd=******
+host=localhost
+database=gsstore
+driver=org.postgresql.Driver
+dbtype=postgis
+```
+
+For more information, refer to the [module documentation](https://docs.geoserver.org/latest/en/user/extensions/wps-jdbc/index.html).
+
+We'd like to thank Ian Turton for developing the module on behalf of GeoSolutions, Alessio Fabiani for providing
+documentation for it, and Andrea Aime for performing the QA and graduation steps.
+
+## WPS Download extension
+
+The WPS download plugin provides processes assisting in the download of large amounts of data,
+allowing to use asynchrounous request, when usage of WFS, WCS or WMS for the same job would lead to HTTP timeouts.
+Also, download limits can be configured to avoid excessively large requests: size in MB, number of features, number
+of animation frames.
+
+In particular, the following processes are available:
+
+*  ``DownloadEstimator``, verifying that a raster/vector download about to be attempted will fit the download limits.
+* ``DownloadProcess``, allowing to download either raster or vector data, reproject and clip them
+* ``DownloadMapProcess``, allows to download a large map matching what is visible on a client (which may be using tiles and display on a multi-screen), eventually dynamically fetching layers from remote WMS servers as well. It's also possible to decorate the final map using the standard [decoration layouts](https://docs.geoserver.org/latest/en/user/services/wms/decoration.html#wms-decorations).
+* ``DownloadAnimationProcess``, allows to build a MP4 movie given a set of layers and times.
+
+(screen snap)
+
+For more information, refer to the [module documentation](https://docs.geoserver.org/latest/en/user/extensions/wps-download/index.html).
+
+Thanks to Alessio, Andrea, Daniele, from GeoSolutions, for developing the extension, and GeoNode/MapStore for testing it in various production environments.
```

Comment:
Thanks to Alessio, Andrea, and Daniele from GeoSolutions for developing the extension, and to GeoNode/MapStore for testing it in various production environments.